modules/oxide-components/src/main/ts/keynav/keyboard/TabbingType.ts

```
@@ -0,0 +1,146 @@
+
+const goForwards: KeyRuleStatelessHandler<FullTabbingConfig> = (component, event, tabbingConfig) => {
+  const navigate = tabbingConfig.cyclic ? ArrNavigation.cycleNext : ArrNavigation.tryNext;
+  return go(component, event, tabbingConfig, navigate);
+};
+
+const goFromPseudoTabstop: KeyRuleStatelessHandler<FullTabbingConfig> = (component, event, tabbingConfig) =>
+  findCurrent(component, tabbingConfig).filter((elem) => !tabbingConfig.useTabstopAt(elem)).bind((elem) => (isFirstChild(elem) ? goBackwards : goForwards)(component, event, tabbingConfig));
+
+const execute = (component: SugarElement<HTMLElement>, _event: Event, tabbingConfig: FullTabbingConfig) => findCurrent(component, tabbingConfig).bind((focused) => tabbingConfig.execute(focused));
+
+const escape = (component: SugarElement<HTMLElement>, _event: Event, tabbingConfig: FullTabbingConfig) => findCurrent(component, tabbingConfig).bind((focused) => tabbingConfig.escape(focused));
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)